spring/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2016 LINE Corporation. All rights reserved.
+  ~ LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+  -->
```

Comment:
We probably should not include this as a part of `armeria-spring` and let our users prepare their own configuration? Would like to know what you think.